cmd/thor/flags.go

```diff
@@ -69,6 +69,14 @@ var (
 		Value: 1000,
 		Usage: "limit the number of logs returned by /logs API",
 	}
+	apiAllowedCredsFlag = cli.BoolFlag{
```

Comment:
can we add these flags to readme with some kind of description / example please ? 😄